### ISSUE TITLE
TLS Based UI

### ISSUE THREAD

**Issue by naman108** [FreeTAKTeam/UI#72]
Given the current security issues for authenticated users, we should add a default TLS option for users to prevent MITM and sniffing attacks.

**Comment by brothercorvo** [comment]
@naman108 push the version to Dev for testing

**Comment by brothercorvo** [comment]
basic functions works, however Rest calls fail at the moment, need more development

**Comment by naman108** [comment]
yup there isn't really a workaround apart supporting a TLS based RestAPI, probably worth creating a new issue in FTS for this.
